package.json

```diff
@@ -12,25 +12,27 @@
   "engines": {
     "node": "^14.17.0 || >=16.0.0"
   },
+  "packageManager": "yarn@1.22.0",
   "scripts": {
-    "prebuild": "npm run -s clean",
-    "build": "npm run build:ts",
+    "prebuild": "yarn clean",
+    "build": "yarn build:ts",
     "build:ts": "tsc --project ./tsconfig.build.json",
-    "clean": "rimraf .nyc_output dist coverage",
+    "clean": "rimraf .nyc_output lib coverage build .svelte-kit svelte.config-dist.js",
     "pretest:base": "cross-env DEBUG=eslint-plugin-svelte*",
     "test": "mocha --require ts-node/register \"tests/src/**/*.ts\" --reporter dot --timeout 60000",
-    "cover": "nyc --reporter=lcov npm run test",
+    "cover": "nyc --reporter=lcov yarn test",
     "debug": "mocha --require ts-node/register/transpile-only \"tests/src/**/*.ts\" --reporter dot --timeout 60000",
     "lint": "eslint .",
     "eslint-fix": "eslint . --fix",
-    "update": "ts-node --transpile-only ./tools/update.ts && npm run format-for-gen-file",
```

Comment:
We can replace `ts-node` with [`tsx`](https://github.com/esbuild-kit/tsx) which is much faster due to `esbuild`, of course it can be a separate PR. If you want I can raise a PR after this PR merged.